### ISSUE TITLE
SQL IN ('string','string') doesn't pull correct results.

### ISSUE THREAD

**Issue by leeish** [cfwheels/cfwheels#564]
Upgrading from 1.1.7 to 1.4.3. We have a findAll call and in the where clause we have:

certifyStatus IN  ('Certified', 'Expired')

However when I dump out the query it results in Cerfitied','Expired as the param. Note the missing quotes at the start and end of the list. So I tried:

certifyStatus IN  (''Certified', 'Expired'')

It put them in there and still bad results. So I changed the query to (certifyStatus = 'Certified' OR certifyStatus = 'Expired') for the time being, but this query worked fine in 1.1.7. Is this intended behavior? How should I be building these queries in 1.4.3.

**Comment by andybellenie** [comment]
Wheels handles the quotes for you, i.e.

certifyStatus IN ('Certified,Expired')

**Comment by leeish** [comment]
Do you know when this was added? ( I tried posting on Google Group but it won't let me. Sorry to post here.) I think this kind of sucks as it forces me to write a bad querey. I get what it's trying to do, but I guess it had to since it allows variables in there. Just wondering when the old way stopped working.

**Comment by leeish** [comment]
I just tried that by the way and it still did the same thing.
![1](https://cloud.githubusercontent.com/assets/3846878/10972584/172257cc-8397-11e5-9884-37ea3901d8b7.PNG)
![2](https://cloud.githubusercontent.com/assets/3846878/10972583/172004fe-8397-11e5-8ad4-4c355a43cac2.PNG)

**Comment by leeish** [comment]
Ah..... I figured it out. In my code there originally, I had a space after the comma, which wheels it looks like saw and knew it should be a string and not a list.

However, it never did/would put in the quotes for me.

So I changed certifyStatus IN ('Certified', 'Expired') to certifyStatus IN ('Certified','Expired'). It looks like the older version just accepted it, which is valid SQL, but I at least I understand what wheels is trying to do that is tell the difference between a list and a string based on the spaces outside the quotes.
